test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -1,5 +1,8 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+Import-Module (Join-Path -Path $PSScriptRoot '..\Microsoft.PowerShell.Security\certificateCommon.psm1') -Force
```

Comment:
```suggestion
Import-Module (Join-Path -Path $PSScriptRoot '..\Microsoft.PowerShell.Security\certificateCommon.psm1')
```

Reply:
Maybe move the needed function to HelpCommon.psm1?